src/main/kotlin/Stores/UserStore.kt

```diff
@@ -33,4 +45,18 @@ object UserStore {
             User.select { User.email.eq(email!!) }.takeIf { !it.empty() }
         }
     }
+    suspend fun getAllUsers(): List<UsersObj> {
```

Comment:
instead of creating this `UsersObj` and returning the `List<UsersObj>` why don't we just return ` User`?

Reply:
ok, that makes sense. Do change it to be singular though, since each object in the list is just one `UserObj`

Reply:
yea updated it in below commit